### ISSUE TITLE
Webtrees trying to fix dates and broke

### ISSUE THREAD

**Issue by Gunnar-GO** [fisharebest/webtrees#3086]
i have some documents that only say the day and month of a date.

if i enter 27.7. or 27.07.____ to generate a placeholder

webtree solves this to born in year 277.

**Comment by vytux-com** [comment]
What happens if you enter the date in the proper format being "27 JUL" without the quotes?

**Comment by fisharebest** [comment]
GEDCOM does not allow incomplete dates such as `27 JUL`.  The year is mandatory.

(To be valid GEDCOM, you should put the incomplete date in a note).

However, webtrees will accept these dates - and it will show them on the anniversaries and calendar.

This issue is about data entry.   GEDCOM dates are written using English keywords and abbreviations.  Since this is not friendly for other languages, webtrees allows many shortcuts.  For example, `dd.mm.yyyy`.  These are listed in the help text.

There is not a shortcut for `dd.mm` (or `mm.dd` for the Americans), because this is not valid GEDCOM.  Also, we cannot know if this is `dd.mm` or `mm.yy`.

When webtrees cannot recognise a date, it will try to find a number, and assume it is a year.
For example, it will find "1950" in "The early 1950s".

In summary, you will need to enter this incomplete/invalid date using GEDCOM format (`27 JUL`).

**Comment by fisharebest** [comment]
The policy of webtrees is that we should be tolerant of invalid data, but we should encourage users to create valid data.

So, I do not think we should add shortcuts to assist in creating invalid dates.

If you want to use them, you will need to enter the data in GEDCOM format - `27 JUL`.
